kolibri/plugins/learn/assets/src/views/LibraryPage/OtherLibraries.vue

```diff
@@ -226,6 +226,22 @@
     props: {
       injectedtr: { type: Function, required: true },
     },
+    watch: {
+      devicesWithChannelsExist: {
+        handler(newValue) {
+          this.$emit('availableNetworkDevices', newValue);
+        },
+        deep: true,
+      },
+    },
+    mounted() {
```

Comment:
I suspect using mounted here might be a little flaky, as we don't actually know that the initial data has been loaded at the point of the component being mounted?

Maybe we can use the `isLoadingChannels` value from the `useDevices` composable to do an initial trigger here rather than mounted? So the first time that goes from `true` to `false` we can then trigger this method.